java/graalvm/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.1.0
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.2.0-b1
```

Comment:
Memo: What's "-b1"? It's written in https://github.com/graalvm/container/pkgs/container/graalvm-ce:

> The version tag defines the level of specificity. It is recommended that the most specific tag be used, e.g., java17-21.3.0 or java17-21.3.0-b1, where the -b1 means the image required a patch and this specific build will never change